File: completable-reactor-runtime/src/main/java/ru/fix/completable/reactor/runtime/internal/gl/GlReactorGraph.java
package ru.fix.completable.reactor.runtime.internal.gl;

import ru.fix.completable.reactor.api.ReactorGraphModel;
import ru.fix.completable.reactor.runtime.ReactorGraph;
import ru.fix.completable.reactor.runtime.gl.Vertex;

import java.util.ArrayList;
import java.util.List;

public class GlReactorGraph<Payload> implements ReactorGraph<Payload> {

    List<Vertex> vertices = new ArrayList<>();







    @Override
    public ReactorGraphModel serialize() {
        throw new UnsupportedOperationException("CR GL does not support serialization, it is deprecated.");
    }

}

File: completable-reactor-runtime/src/main/kotlin/ru/fix/completable/reactor/runtime/internal/gl/GlReactorGraph.kt
package ru.fix.completable.reactor.runtime.internal.gl

import ru.fix.completable.reactor.api.ReactorGraphModel
import ru.fix.completable.reactor.runtime.ReactorGraph
import ru.fix.completable.reactor.runtime.gl.Vertex
import java.util.*

class GlReactorGraph<Payload> : ReactorGraph<Payload> {

    var vertices: List<Vertex> = ArrayList()
    var startPoint: MutableList<Vertex> = ArrayList()

    override fun serialize(): ReactorGraphModel {
        throw UnsupportedOperationException("CR GL does not support serialization, it is deprecated.")
    }
}
